docs/swap.md

```diff
@@ -160,13 +160,7 @@ make start-cron
 
 Tracing
 =======
-In Honeycomb you can query the following:
-
-```
-swap.error exists
-swap.submitted exists
-swap.success exists
-```
+In Honeycomb you can query for `swap`
```

Comment:
do you mean `name: swap`?

Reply:
yeah actually `code.namespace = services.swap`. i think i remember Justin mentioning to use`addAttributesToCurrentSpan` instead of creating a new one, due to cost.